cms/djangoapps/contentstore/signals/handlers.py

```diff
@@ -159,11 +160,46 @@ def listen_for_course_publish(sender, course_key, **kwargs):  # pylint: disable=
     transaction.on_commit(lambda: emit_catalog_info_changed_signal(course_key))
 
 
+def _determine_producer_config_for_signal_and_topic(signal, topic):
+    """
+    Utility method to determine the setting for the given signal and topic in EVENT_BUS_PRODUCER_CONFIG
+
+    Records to New Relic for later analysis.
+
+    Parameters
+        signal (OpenEdxPublicSignal): The signal being sent to the event bus
+        topic (string): The topic to which the signal is being sent
+
+    Returns
+        True if the signal is enabled for that topic in EVENT_BUS_PRODUCER_CONFIG
+        False if the signal is explicitly disabled for that topic in EVENT_BUS_PRODUCER_CONFIG
+        None if the signal/topic pair is not present in EVENT_BUS_PRODUCER_CONFIG
+    """
+    event_type_producer_configs = getattr(settings, "EVENT_BUS_PRODUCER_CONFIG",
+                                          {}).get(signal.event_type, {})
+    topic_config = event_type_producer_configs.get(topic, {})
+    topic_setting = topic_config.get('enabled', None)
+    if topic_setting is True:
+        set_custom_attribute('producer_config_setting', 'True')
```

Comment:
What about something like?:
```
- set_custom_attribute(f'producer_config_{signal}_{topic}', topic_setting)
```

Reply:
If I use `topic_setting,` will it report Nones correctly?

Reply:
Also I feel like including the signal in the attribute name will make the names pretty unwieldy. I'm not sure it's necessary

Reply:
In terms of "unwieldy", it would mean that the attribute would need to be combined with the New Relic transaction name to know where there is an issue. If we have 2 events that might be sent from the same transaction, one would get overwritten by the other. Note: this is temporary, so you decide.

Reply:
[fixed]